src/jobs/jobs.service.ts

```
@@ -61,8 +63,103 @@ export class JobsService {
 
     private readonly blockchainService: BlockchainService,
     private readonly currencyConversionService: CurrencyConversionService,
+
+    @InjectRepository(Escrow)
+    private readonly escrowRepository: Repository<Escrow>,
+    private readonly blockchainService: BlockchainService,
```

Biome (1.9.4)

[error] Duplicate parameter name.

The parameter overrides a preceding parameter by using the same name.

(lint/suspicious/noDuplicateParameters)